### PR TITLE
Updates to magenta_docker

### PR DESCRIPTION
**Pull request by ana-simionescu**
- Added documentation on hyperparameter tuning, tensorboard and user interaction script
- Added installation of cloudml-hypertune in dockerfile needed for hyperparameter tuning
- Added the user_interaction script
- Added a default config file for hyperparameter tuning

### CODE REVIEW

File: magenta_docker/user_interaction.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+import os
+
+# Determining path for data retrieval
+data_path = raw_input('Insert a path for data retrieving. '
+'This must be a GCS Bucket:\n')
```
Comment (marmolejo) [inline magenta_docker/user_interaction.py]: Here and below: This line should come under `'Insert`. Have you processed this file agains py_lint?

File: magenta_docker/user_interaction.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+if steps_per_summary == "":
+    steps_per_summary = "300"
+
+early_stop_loss_value = raw_input('\nInsert the early stop loss value '
+'or skip for the default value (5): ')
+if early_stop_loss_value == "":
+    early_stop_loss_value = "5"
+
+# Submit the job on AI Platform
+print("Submitting the job on AI Platform")
```
Comment (marmolejo) [inline magenta_docker/user_interaction.py]: I think I would write this script in a more structured way. I would create a function to gather all the input and another function to run the commands.

**Review by marmolejo — CHANGES_REQUESTED** [review]

File: magenta_docker/user_interaction.py (hunk 1 above)
Comment (ana-simionescu) [reply magenta_docker/user_interaction.py]: No, I will do it right now and make the changes.

File: magenta_docker/user_interaction.py (hunk 2 above)
Comment (ana-simionescu) [reply magenta_docker/user_interaction.py]: Okay, I will do this.

**Review by marmolejo — APPROVED** [review]
Looks good!

**Pull request merged**